compiler/parser/parser.peg

```diff
@@ -478,6 +479,23 @@ SampleExpr
   = _ lval:Lval { RETURN(lval)}
   / "" { RETURN(MAP("kind":"ID", "name":"this")) }
 
+LoadOp
+    = "load" _ pool:PoolNameString author:AuthorArg? message:MessageArg? meta:MetaArg? branch:BranchSpecification? {
+        RETURN(MAP("kind": "Load", "pool": pool, "author": author, "message": message, "meta": meta, "branch": branch))
+    }
+
+AuthorArg
+  = _ "author" _ val:QuotedString { RETURN(val) }
+
+MessageArg
+  = _ "message" _ val:QuotedString { RETURN(val) }
+
+MetaArg
+  = _ "meta" _ val:QuotedString { RETURN(val) }
+
+BranchSpecification
+  = _ "@" _ branch:QuotedString { RETURN(branch) }
```

Comment:
1. This is pretty similar to PoolCommit, so call it PoolBranch.
2. In keeping with PoolCommit, don't require (or allow) spaces around "@".
3. Allow PoolIdentifier in addition to QuotedString so we can say `pool@main` instead of having to say `pool@"main"`.
```suggestion
PoolBranch
  = "@" branch:(PoolIdentifier / QuotedString) { RETURN(branch) }
```